### PR TITLE
Post release/merge 6.1.0 into development

### PR DESCRIPTION
**Pull request by ethan605**
Merge `master` including `release/6.1.0` into `development`

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1218-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1218-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1218-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1218-pr-onfido-sdk-ui-onfido.surge.sh

File: demo/fiddle/demo.details [hunk 1]
```diff
@@ -3,6 +3,6 @@
  authors:
    - Onfido Team
  resources:
-   - https://assets.onfido.com/web-sdk-releases/6.0.1/onfido.min.js
-   - https://assets.onfido.com/web-sdk-releases/6.0.1/style.css
+   - https://assets.onfido.com/web-sdk-releases/6.1.0-rc.1/onfido.min.js
+   - https://assets.onfido.com/web-sdk-releases/6.1.0-rc.1/style.css
```
Comment (kopijunkie) [inline demo/fiddle/demo.details:7]: These should be `6.1.0`

**Review by kopijunkie — CHANGES_REQUESTED** [review]
jsFiddle version needs to be updated

**Review by kopijunkie — APPROVED** [review]

**Review by stefaniacardenas — APPROVED** [review]
LGTM

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1218-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1218-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1218-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**